cmd/skopeo/inspect.go

```diff
@@ -135,9 +148,19 @@ func (opts *inspectOptions) run(args []string, stdout io.Writer) (retErr error)
 		}, opts.retryOpts); err != nil {
 			return errors.Wrapf(err, "Error reading OCI-formatted configuration data")
 		}
-		err = json.NewEncoder(stdout).Encode(config)
+		if report.IsJSON(opts.format) || opts.format == "" {
```

Comment:
The `IsJSON` case is, AFAICS, undocumented — and especially for the `{{json}}` case, which looks very much like a valid Go template, surprising. Why is it useful?